src/components/Exchange/FeesTooltip.tsx

```diff
@@ -0,0 +1,85 @@
+import { t, Trans } from "@lingui/macro";
+import ExternalLink from "components/ExternalLink/ExternalLink";
+import StatsTooltipRow from "components/StatsTooltip/StatsTooltipRow";
+import Tooltip from "components/Tooltip/Tooltip";
+import { getConstant } from "config/chains";
+import { BigNumberish } from "ethers";
+import { useChainId } from "lib/chains";
+import { USD_DECIMALS } from "lib/legacy";
+import { formatAmount, formatAmountFree } from "lib/numbers";
+
+type Row = {
+  label: string;
+  value: string;
+};
+
+function getExecutionFeeStr(chainId, executionFee, executionFeeUsd) {
+  const nativeTokenSymbol = getConstant(chainId, "nativeTokenSymbol");
+
+  if (!nativeTokenSymbol || !executionFee || !executionFeeUsd) {
+    return "...";
+  }
+
+  const formattedExecutionFee = formatAmountFree(executionFee, 18, 4);
+  const formattedExecutionFeeUsd = formatAmount(executionFeeUsd, USD_DECIMALS, 2);
+  return `${formattedExecutionFee} ${nativeTokenSymbol} ($${formattedExecutionFeeUsd})`;
+}
+
+type Props = {
+  totalFees: string;
+  fundingFee: string;
+  positionFee: string;
+  depositFee: string;
+  swapFee: string;
+  executionFees: { fee: BigNumberish; feeUSD: BigNumberish };
+  isOpening?: boolean;
+};
+
+function FeesTooltip({
```

Comment:
mm, maybe FeesTooltip will accept BigNumbers instead of strings and format fee strings inside?

Then the component will be easier to use and there will be no duplicate logic in the SwapBox, PositionEditor, etc

Reply:
Done!